src/App.js

```diff
@@ -1,29 +1,13 @@
-const todoList = [
-  {
-    id: '1',
-    title: 'Record Music'
-  },
-  {
-    id: '2',
-    title: 'Write Code'
-  },
-  {
-    id: '3',
-    title: 'Connect Wires'
-  }
-]
+import { React } from "react";
+import ToDoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
 
 function App() {
   return (
-    <div className="App">
+    <div>
       <h1>ToDo List</h1>
-      <ul>
-        {todoList.map(item => {
-          return <li key={item.id}>
-            <span>{item.title}</span>
-          </li>
-        })}
-      </ul>
+      {AddTodoForm()}
```

Comment:
You'll want to use your components in this syntax `<AddToDoForm />` rather than `{AddTodoForm()}`. You may run into some issues with that later otherwise, but it reads more semantically as to the intentional of the component.